test/testdata/cfg/return_expr_void.rb

```diff
@@ -0,0 +1,8 @@
+# typed: true
+class C
+  extend T::Sig
+  sig {void}
+  def foo
+    return 3 # error: `C#foo` returns `void` but contains an explicit `return <expression>`
+  end
+end
```

Comment:
I'm sure this is tested elsewhere in the suite, but can you add a test here that makes it clear that `return` by itself doesn't cause this error?